src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1928,24 +1877,6 @@ public boolean match(final Unit unit) {
     };
   }
 
-  public static Match<Unit> unitHasEnoughTransportSpaceLeft(final int spaceNeeded) {
```

Comment:
- YAGNI principle
- It's one more untested thing
- It's one more thing to update when refactoring, if someone spends 20 minutes figuring what this does, regression testing it, all to then figure it it was never used, they would not be happy. Sample code can go into many other places other than the source code that is compiled and coupled with many other parts of the code. Killing dead code is one of the fastest ways to reduce code coupling.

So with all respect, unless there is a needed concrete use case, in interest of simplifying this code base, the default answer should be to go with less code, simpler code, and YAGNI.

With that said, if you really want this code around, I can leave it an as a private static method with an unused dead code compiler warning.